.dprint.jsonc

```diff
@@ -28,7 +28,6 @@
         "./types/react-native",
         "./types/recurly__recurly-js",
         "./types/sawtooth-sdk",
-        "./types/webextension-polyfill",
```

Comment:
is this accurate? I still see at least one file that says it's generated automatically. Does the generation process now format with dprint?

Reply:
Yes, the generation process now uses dprint: https://github.com/Lusito/webextension-polyfill-ts/blob/3f2243a599988f93fd83159a6d54fb6e8f41baac/package.json#L25